scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -0,0 +1,410 @@
+                                continue
+
+                            dest_dir = OUTPUT_DIR / variant
+                            dest = dest_dir / asset_name
+                            print(
+                                f"    Downloading {platform.value}/{arch.value}: {asset_name}"
+                            )
+                            try:
+                                download_asset(
+                                    client, download_url, dest
+                                )
+                            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)